### PR TITLE
add yacman recipe

### PR DESCRIPTION
**Pull request by michalstolarczyk**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/yacman/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "yacman" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "37cd49463880f4c9cfee0dc8280bc893fc3bf392615d12b35c97609def71071c"
```
Comment (marcelotrevisani) [inline recipes/yacman/meta.yaml]: Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 37cd49463880f4c9cfee0dc8280bc893fc3bf392615d12b35c97609def71071c
```

File: recipes/yacman/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/yacman/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/yacman/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - yacman
+
+about:
+  home: "https://github.com/databio/yacman"
+  license: "BSD"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
+  summary: "A standardized configuration object for reference genome assemblies"
+  doc_url: ""
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/yacman/meta.yaml]: Could you please remove those empty ``doc_url`` and ``dev_urk`` ?

File: recipes/yacman/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "37cd49463880f4c9cfee0dc8280bc893fc3bf392615d12b35c97609def71071c"
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/yacman/meta.yaml]: I think this recipe could be ``noarch``
```suggestion
  number: 0
  noarch: python
```

File: recipes/yacman/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - attmap >=0.12.9
+    - oyaml
+    - pip
+    - python
+    - pyyaml >=3.13
+    - ubiquerg >=0.4.9
```
Comment (marcelotrevisani) [inline recipes/yacman/meta.yaml]: In the host section, I think you just need ``python`` and ``pip``

File: recipes/yacman/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - yacman
+
+about:
+  home: "https://github.com/databio/yacman"
+  license: "BSD"
```
Comment (marcelotrevisani) [inline recipes/yacman/meta.yaml]:
```suggestion
  license: "BSD-2-Clause"
```

File: recipes/yacman/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - attmap >=0.12.9
+    - oyaml
+    - python
+    - pyyaml >=3.13
+    - ubiquerg >=0.4.9
```
Comment (marcelotrevisani) [inline recipes/yacman/meta.yaml]: This package does not exist in ``conda-forge``, you will need to package ``ubiquerg`` and ``attmap`` first

Comment (michalstolarczyk) [reply recipes/yacman/meta.yaml]: PRs are opened for these

**Comment by michalstolarczyk** [comment]
waiting for #10035

**Comment by michalstolarczyk** [comment]
@marcelotrevisani it this ready?

File: recipes/yacman/meta.yaml (hunk 7 above)
Comment (michalstolarczyk) [reply recipes/yacman/meta.yaml]: both exist in `conda-forge` now

**Pull request merged**